### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Bleak671**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+            <div class="header_menu">
+                <image class="logo" src="images/hdr_logo.png"></image>
+                <a class="text_white menu_item font_raleway_sb font_size_tiny" href="#">HOME</a>
+                <a class="text_white menu_item font_raleway_sb font_size_tiny" href="#">SERVICES</a>
+                <a class="text_white menu_item font_raleway_sb font_size_tiny" href="#">PORTFOLIO</a>
+                <a class="text_white menu_item font_raleway_sb font_size_tiny" href="#">BLOG</a>
+                <a class="text_white menu_item font_raleway_sb font_size_tiny" href="#">CONTACT US</a>
+                <a class="text_white menu_item_login menu_item font_raleway_sb font_size_tiny" href="#">LOGIN</a>
+            </div>
+            <image src="images/hdr_dividor.png" width="100%"></image>
```
Comment (e44alex) [inline index.html]: this could be done without resource, using only html tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+            <div class="header_content">
+                <div class="header_text">
+                    <h1 class="text_white font_museo font_size_huge">FLAT + FREE = FLABY</h1>
+                    <p class="text_white font_raleway font_size_medium">A Modern Flat design corporate template for free download with all great features and trendy look.</p>
+                    <div class="header_buttons">
```
Comment (e44alex) [inline index.html:41]: you need to make this clickable, even though it has no action on click

File: index.html [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+            <p class="text_gray font_raleway font_size_tiny text_margin_bottom">Contact Us</p>
+        </div>
+    </div>
+
+    <image class="dividor" src="images/ftr_dividor.png"></image>
+    <footer>
+        <p class="text_gray font_raleway">© Copyright 2017 | All rights reserved | Template Designed By <b>Sourabh Baura</b></p>
+    </footer>
+
+    <script>
```
Comment (e44alex) [inline index.html]: please do not put JS into html files directly. extract it and link script file.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+            <p class="text_gray font_roboto font_size_tiny text_margin_bottom">For latest newsletter</p>
+            <p class="text_mildblack font_roboto font_size_medium text_margin_bottom">SUBSCRIBE NOW</p>
+            <image src="images/dividor.png" height="2px" width="30px"></image>
+        </div>
+        <div class="subscribe_blocks">
+            <div class="subscribe_block">
+                <input class="text_gray font_raleway_b font_size_tiny" value="Nicky Moa">
+            </div>
+            <div class="subscribe_block bg_mild_blue">
+                <input class="text_gray font_raleway_b font_size_tiny bg_mild_blue" value="Example@gmail.com">
```
Comment (e44alex) [inline index.html]: it looks like your intended to make placeholder. input tag has an attribute for it, do not use value.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,430 @@
...
+
+footer {
+    width: 66%;
+    margin-left: 17%;
+    margin-top: 30px;
+    margin-bottom: 70px;
+}
+
+
+/*Properties of header block*/
```
Comment (e44alex) [inline style.css]: why don't you split such huge css file. I guess, it would be easier to work with many small files

File: style.css [hunk 6]
```diff
@@ -0,0 +1,430 @@
...
+
+input {
+    border-width: 0px;
+    outline: none;
+}
+
+footer {
+    width: 66%;
+    margin-left: 17%;
+    margin-top: 30px;
```
Comment (e44alex) [inline style.css]: pixels are good as absolute value. but not the best thing when making margins, paddings, height, width, etc. Find out which relative values you can use and apply where possible.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,286 @@
...
+<head>
+    <meta charset="utf-8" />
+    <title>Flaby</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+    <div class="header">
+        <div class="header_left_space">
+            <image class="header_image" src="images/hdr_headphones.png" width="90%"></image>
```
Comment (e44alex) [inline index.html:21]: It looks like this thing overlays your header 
image

File: index.html (hunk 2 above)
Comment (e44alex) [reply index.html:41]: also there is something with alignment. it floats overs blue block on small screens
image

File: index.html (hunk 1 above)
Comment (e44alex) [reply index.html]: as well as other dividers, btw

File: index.html [hunk 8]
```diff
@@ -0,0 +1,286 @@
...
+    </div>
+
+    <div class="comment">
+        <div class="our_serv_head">
+            <p class="text_gray font_roboto font_size_tiny text_margin_bottom">Get know About</p>
+            <p class="text_mildblack font_roboto font_size_medium text_margin_bottom">WHAT CUSTOMER SAY</p>
+            <image src="images/dividor.png" height="2px" width="30px"></image>
+        </div>
+        <div class="comment_content">
+            <div class="comment_text">
```
Comment (e44alex) [inline index.html:152]: cards are cropped on small screen
image

File: style.css [hunk 9]
```diff
@@ -0,0 +1,430 @@
...
+    padding-bottom: 15px;
+    align-self: center;
+}
+
+.menu_item_login {
+    border-radius: 10px;
+    background-color: rgba(70, 70, 70, 0.2);
+}
+
+@font-face {
```
Comment (e44alex) [inline style.css]: there is also an issue with custom fonts. this thing points to local installation. So, as a result, when I launched you page I didn't see them

File: index.html [hunk 10]
```diff
@@ -0,0 +1,286 @@
...
+        </div>
+    </div>
+    <image class="dividor" src="images/ftr_dividor.png"></image>
+
+    <div class="links">
+        <div class="links_column_first">
+            <image class="logo" src="images/etc_logo.png"></image>
+            <p class="text_gray text_margin_bottom text_margin_top font_raleway font_size_tiny">Flaby is a free flat template and available for free downloads but please don't use for sell</p>
+            <div class="links_social">
+                <image src="images/etc_facebook.png"></image>
```
Comment (e44alex) [inline index.html]: should be links to social media. Even though link reference not provided.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,286 @@
...
+                <image src="images/etc_google.png"></image>
+                <image src="images/etc_twitter.png"></image>
+            </div>
+        </div>
+        <div class="links_column">
+            <p class="text_mildblack font_raleway_b font_size_little text_margin_bottom">Useful Link</p>
```
Comment (e44alex) [inline index.html]: links please

File: index.html [hunk 12]
```diff
@@ -0,0 +1,286 @@
...
+                <image class="dot" src="images/ftr_point.png">
+                    </imgage>
+                    <p class="text_gray text_margin_top text_margin_bottom custom_text font_raleway font_size_tiny">Free vector shape and mockup by great designer.</p>
+            </div>
+            <div class="custom_content_row">
+                <image class="dot" src="images/ftr_point.png">
+                    </imgage>
+                    <p class="text_gray text_margin_top text_margin_bottom custom_text font_raleway font_size_tiny">Full one page flat and modern trend following goods.</p>
+            </div>
+            <a class="text_white custom_button bg_blue font_raleway_b font_size_tiny" href="#">READ MORE</a>
```
Comment (e44alex) [inline index.html:99]: This also an issue when viewing on small screen
image

Comment (e44alex) [reply index.html:99]: Also. href="#" navigates to the top of the page. Let's see if you can make it navigate to the other element :)

File: style.css [hunk 13]
```diff
@@ -0,0 +1,430 @@
+/*Common props for object types*/
+
```
Comment (e44alex) [inline style.css]: This is a common practice to put styles in a separate folder, not in the root dir

File: index.html [hunk 14]
```diff
@@ -0,0 +1,286 @@
...
+        <div class="custom_photo">
+            <image src="images/ftr_mockup.png" width="80%"></image>
+        </div>
+        <div class="custom_content">
+            <div class="custom_content_header">
+                <p class="text_gray font_raleway_sb font_size_little text_margin_bottom">We have great features with</p>
+                <p class="font_raleway_sb font_size_big  text_margin_bottom text_mildblack">Unlimited Customization Features</p>
+                <p class="text_gray font_raleway font_size_little">There are many variations of passages of Lorem Ipsum available, the majorityalteration make difficult</p>
+            </div>
+            <div class="custom_content_row">
```
Comment (e44alex) [inline index.html]: Looks like you are using same pic 4 times. I see an opportunity of using list (li) and styling dots. DRY.

File: index.html (hunk 11 above)
Comment (e44alex) [reply index.html]: Here as well. So many repeating styles. You can style list item 1 time and then reuse it all the way down.

File: index.html (hunk 2 above)
Comment (Bleak671) [reply index.html:31]: buttons now clickable, but floats anyway

File: index.html [hunk 15]
```diff
@@ -0,0 +1,250 @@
...
+                </a>
+                <a href="https://twitter.com">
+                    <image src="images/etc_twitter.png"></image>
+                </a>
+            </div>
+        </div>
+        <div class="links_column">
+            <a class="text_mildblack font_raleway_b font_size_little text_margin_bottom" href="#">Useful Link</a>
+            <a class="text_gray font_raleway font_size_tiny text_margin_top text_margin_bottom" href="#">Resource</a>
+            <a class="text_gray font_raleway font_size_tiny text_margin_bottom" href="#">Downloads</a>
```
Comment (e44alex) [inline index.html]: here repeating classes. extract to 1 common and reuse it

File: index.html [hunk 16]
```diff
@@ -0,0 +1,250 @@
...
+                <p class="text_gray font_raleway font_size_tiny">There are many variations of passages of Lorem Ipsum available, the majorityalteration make difficult</p>
+            </div>
+        </div>
+    </div>
+
+    <div class="comment" id="head5">
+        <div class="our_serv_head">
+            <p class="text_gray font_roboto font_size_tiny text_margin_bottom">Get know About</p>
+            <p class="text_mildblack font_roboto font_size_medium text_margin_bottom">WHAT CUSTOMER SAY</p>
+            <image src="images/dividor.png" height="2px" width="30px"></image>
```
Comment (e44alex) [inline index.html]: hr

File: index.html [hunk 17]
```diff
@@ -0,0 +1,250 @@
...
+            </ul>
+            <a class="text_white custom_button bg_blue font_raleway_b font_size_tiny" href="#">READ MORE</a>
+        </div>
+    </div>
+
+    <div class="our_serv" id="head4">
+        <div class="our_serv_head">
+            <p class="text_gray font_roboto font_size_tiny text_margin_bottom">View Our Latest</p>
+            <p class="text_mildblack font_roboto font_size_medium text_margin_bottom">OTHER SERVICES</p>
+            <image src="images/dividor.png" height="2px" width="30px"></image>
```
Comment (e44alex) [inline index.html]: hr

File: index.html [hunk 18]
```diff
@@ -0,0 +1,250 @@
...
+            <image src="images/ftr_mockup.png" width="80%"></image>
+        </div>
+        <div class="custom_content">
+            <div class="custom_content_header">
+                <p class="text_gray font_raleway_sb font_size_little text_margin_bottom">We have great features with</p>
+                <p class="font_raleway_sb font_size_big  text_margin_bottom text_mildblack">Unlimited Customization Features</p>
+                <p class="text_gray font_raleway font_size_little">There are many variations of passages of Lorem Ipsum available, the majorityalteration make difficult</p>
+            </div>
+            <ul>
+                <li class="text_gray custom_text font_raleway font_size_tiny">More wordpress features with great responsive design.</li>
```
Comment (e44alex) [inline index.html]: extract multiple classes to one class for this list item

File: index.html [hunk 19]
```diff
@@ -0,0 +1,250 @@
...
+        <div class="header_right_space">
+            <image class="header_image header_image_earphones" src="images/hdr_earphones.png" width="55%"></image>
+        </div>
+    </div>
+
+    <div class="our_serv" id="head2">
+        <div class="our_serv_head">
+            <p class="text_gray font_roboto font_size_little">Welcome to</p>
+            <p class="font_roboto font_size_medium text_margin_bottom text_margin_top text_mildblack">OUR SERVICES</p>
+            <image src="images/dividor.png" height="2px" width="30px"></image>
```
Comment (e44alex) [inline index.html]: change to hr

File: index.html [hunk 20]
```diff
@@ -0,0 +1,250 @@
...
+    <div class="header" id="head1">
+        <div class="header_left_space">
+            <image class="header_image" src="images/hdr_headphones.png" width="90%"></image>
+            <image class="header_image" src="images/hdr_pencil.png" width="70%"></image>
+        </div>
+        <div class="header_center">
+            <div class="header_menu">
+                <image class="logo" src="images/hdr_logo.png"></image>
+                <a class="text_white menu_item font_raleway_sb font_size_tiny" href="#">HOME</a>
+                <a class="text_white menu_item font_raleway_sb font_size_tiny" href="#">SERVICES</a>
```
Comment (e44alex) [inline index.html]: repeating classes. could be refactored

**Review by e44alex — APPROVED** [review]

**Pull request merged**